apis/python/tests/test_dense_nd_array.py

```diff
@@ -583,3 +580,39 @@ def test_read_result_order(tmp_path):
             DeprecationWarning, match="The use of 'result_order=\"auto\"' is deprecated"
         ):
             assert np.array_equal(A.read(result_order="auto"), data)
+
+
+def test_slice_with_offset(tmp_path):
+    uri = tmp_path.as_posix()
```

Comment:
tests should use a fixture for temp paths (otherwise these files don't get cleaned up reliably) - see sc-61088

Reply:
sorry, this is a bogus comment from me - please ignore.  You are doing the right thing already!

Reply:
Yeah, this was completely _my_ brain segfault.  Your code is great!

Reply:
@bkmartinjr is `tmp_path` not a fixture for temp paths?

Reply:
It is.  See my two previous comments apologizing for my bogon!  Super sorry

Reply:
https://docs.pytest.org/en/stable/how-to/tmp_path.html